src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,128 @@
+---
+linkTitle: Creating an app catalog
+title: Creating an app catalog
+description: How to create a custom app catalog for use with app platform and push helm charts to it.
+weight: 40
+menu:
+  main:
+    parent: app-platform
+last_review_date: 2021-12-01
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+user_questions:
+  - How can I create an app catalog for my apps?
+  - How can I serve an app catalog using GitHub Pages?
+  - How can I create a catalog CR for my app catalog?
+  - How can I push an app to my app catalog?
+---
+
+# Creating an app catalog
+
+## Overview
+
+An app catalog is a collection of apps that can be deployed using the Giant Swarm App Platform.
+We fully support Helm and an app catalog is also a Helm [Chart Repository](https://helm.sh/docs/topics/chart_repository/).
+Each app catalog has its own [catalog]({{< relref "/ui-api/management-api/crd/catalogs.application.giantswarm.io.md" >}})
+CR in the management cluster.
+
+The app catalog contains a tarball for each version of an app that has been published.
+As well as an index.yaml and other metadata files that are used to generate
+metadata that is stored in the management cluster in [app catalog entry]({{< relref "/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}})
+CRs.
+
+The catalog needs to be served over HTTP and there are [multiple options](https://helm.sh/docs/topics/chart_repository/#hosting-chart-repositories)
+including GitHub Pages or tools like Harbor or ChartMuseum which run in a
+Kubernetes cluster. At Giant Swarm we use GitHub Pages and this is what we will
+cover in this guide.
+
+## Creating an app catalog using GitHub Pages
+
+First you should choose a name for your catalog. We recommend using the suffix
+`-catalog` to make it clear this repository hosts an app catalog.
+
+Create the git repository in GitHub and enable [GitHub Pages](https://docs.github.com/en/pages/quickstart)
+for the `main` branch.
+
+Initialize the empty helm index.yaml with `helm repo index .` and commit it to
+the repository.
+
+## Pushing an app to an app catalog
+
+TODO
```

Comment:
@piontec Just to confirm. 😅 Do you agree we should use ABS for this? 

PTAL at the questions below.

Reply:
Yes, definitely. We should make it our 'standard' for building charts.